java_tools/gcc_map_reader/build.gradle

```diff
@@ -3,6 +3,8 @@ plugins {
     id 'application'
 }
 
+mainClassName = 'com.rusefi.Launcher'
```

Comment:
I do not get this, how would this change what?

Reply:
Well, it makes startScripts task work and doesn't break the run task which originally launches the rusefi_console.jar